app/views/petitions/_open_petition_show.html.erb

```diff
@@ -18,13 +18,15 @@
   <div class="signature-count-graph" aria-hidden="true">
     <span class="signature-count-current" style="width: <%= signatures_threshold_percentage(petition) %>"></span>
   </div>
-  <p class="signature-count-goal">
-    <% if petition.signature_count < Site.threshold_for_response %>
-      <%= number_with_delimiter(Site.threshold_for_response) %>
-    <% else %>
-      <%= number_with_delimiter(Site.threshold_for_debate) %>
-    <% end %>
-  </p>
+  <% unless petition.debate_threshold_reached_at? || petition.scheduled_debate_date? || petition.debate_outcome_at? %>
```

Comment:
We have a couple of tests, but not enough to cover all the scenarios. Probably outside of my XP level